### PR TITLE
Implement Aether Landing page

### PR DESCRIPTION
**Pull request by umarphaarook**
Implement an app that serves as a UI for accessing different running servives.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 This application is used to configure Keycloak+Kong installations,
 locally or as part of a cluster.
 
+## Aether Landing Page
+
+An app that serves as a UI for accessing different running servives.
```
Comment (obdulia-losantos) [inline README.md]: `servives` ???

File: aether-landing-page/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  },
+  plugins: [
+    'react'
+  ],
+  rules: {
+    "react/prop-types": 0
+  }
+}
```
Comment (obdulia-losantos) [inline aether-landing-page/.eslintrc.js]: We don't use `eslint`, we use `standard` taht doesn't require any config file like this one :grimacing:

File: aether-landing-page/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Aether Landing Page
```
Comment (obdulia-losantos) [inline aether-landing-page/README.md]: `# Aether Landing Page`  :wink:

File: aether-landing-page/package.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    "name": "eHealth Africa",
+    "email": "info@ehealthafrica.org",
+    "url": "https://ehealthafrica.org"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "yarn lint && jest",
+    "test:watch": "yarn test --watch",
+    "lint": "eslint \"**/*.{js,jsx}\""
```
Comment (obdulia-losantos) [inline aether-landing-page/package.json]: as I mentioned above `standard` take a look at `aether-ui` or `gather` :nerd_face:

File: aether-landing-page/src/utils/constants.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ * under the License.
+ */
+
+export const DEFAULT_LOCALE = 'en'
+
+export const SAMPLE_ABOUT = `
+  Lorem ipsum dolor sit amet, 
+  consectetur adipiscing elit, 
+  sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+  `
```
Comment (obdulia-losantos) [inline aether-landing-page/src/utils/constants.jsx]: :flushed:

File: aether-landing-page/package.json [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    "test:watch": "yarn test --watch",
+    "lint": "standard './src/**/*.{js,jsx}'"
+  },
+  "dependencies": {
+    "bootstrap": "~4.3.1",
+    "react": "~16.12.0",
+    "react-dom": "~16.12.0",
+    "react-intl": "~3.6.2",
+    "react-scripts": "3.2.0",
+    "react-test-renderer": "~16.12.0"
```
Comment (obdulia-losantos) [inline aether-landing-page/package.json]: Isn't this a dev dependency? :thinking:

File: aether-landing-page/src/utils/constants.jsx (hunk 5 above)
Comment (umarphaarook) [reply aether-landing-page/src/utils/constants.jsx]: @adamvert to provide the info about the services.. ℹ️

File: aether-landing-page/src/css/base/_fonts.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+// sass-lint:disable no-url-domains no-url-protocols
+
+/* Google Fonts */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,400i,600');
+@import url('https://fonts.googleapis.com/css?family=Fira+Mono');
+
+
+/* Font Awesome 5  https://fontawesome.com/ */
+@import url('https://use.fontawesome.com/releases/v5.11.2/css/fontawesome.css');
+@import url('https://use.fontawesome.com/releases/v5.11.2/css/solid.css');
```
Comment (obdulia-losantos) [inline aether-landing-page/src/css/base/_fonts.scss]: The current version is `5.12.0`

**Comment by shawnsarwar** [comment]
@umaroolfaruk What's the status on this?

**Comment by obdulia-losantos** [comment]
From my side I still need to run this locally and check that it works as expected. After that we need to include the script that pushes the app to a google cloud bucket and serve that behind our kong instance.

**Comment by umarphaarook** [comment]
> From my side I still need to run this locally and check that it works as expected. After that we need to include the script that pushes the app to a google cloud bucket and serve that behind our kong instance.

File: aether-landing-page/src/utils/index.jsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+ * @param {string} str
+ */
+export const capitalize = (str) => `${str[0].toUpperCase()}${str[1] ? str.slice(1) : ''}`
+
+/**
+ * Return a list of services for a given tenant
+ *
+ * @param {string} tenant
+ */
+export const getServices = (tenant = '', origin = window.location.origin) => [
```
Comment (lordmallam) [inline aether-landing-page/src/utils/index.jsx]:
```suggestion
export const getServices = (tenant = '', origin = window.location.origin) => {
    const tenant_path = tenant ? `${tenant}/` : ''
    return [
        { name: GATHER, icon: gatherIcon, link: `${origin}/${tenant_path}${GATHER}` },
        { name: KIBANA, icon: kibanaIcon, link: `${origin}/${tenant_path}${KIBANA}/kibana-app` },
        { name: '' },
        { name: AETHER, icon: aetherIcon, link: `${origin}/${tenant_path}${AETHER}` },
        { name: KERNEL, link: `${origin}/${tenant_path}${KERNEL}` },
        { name: ODK, link: `${origin}/${tenant_path}${ODK}` }
    ]
}
```

Comment (lordmallam) [reply aether-landing-page/src/utils/index.jsx]: @umaroolfaruk ☝️ url fix for when a tenant is not supplied

**Review by lordmallam — APPROVED** [review]

**Comment by sunny775** [comment]
Hi @umaroolfaruk 
I can't send you message on twitter.
Can we chat on this slack channel:
https://join.slack.com/t/jsgeeksworkspace/shared_invite/zt-ff43op8i-ADkOc1uUgEICNrh69N~oXA

**Comment by sunny775** [comment]
Hi @umaroolfaruk 
I can't send you message on twitter.
Can we chat on this slack channel:
https://join.slack.com/t/jsgeeksworkspace/shared_invite/zt-ff43op8i-ADkOc1uUgEICNrh69N~oXA

**Pull request merged**